etc/config/c++.amazon.properties

```
@@ -2750,6 +2750,13 @@ libs.spy.versions.v004.alias=v003
 libs.spy.versions.v100.version=v1.0.0
 libs.spy.versions.v100.path=/opt/compiler-explorer/libs/spy/1.0.0/include
 
+libs.stdexec.name=std::execution
+libs.stdexec.description=The proposed C++ framework for asynchronous and parallel programming
+libs.stdexec.versions=trunk
+libs.stdexec.url=https://github.com/NVIDIA/stdexec
+libs.stdexec.versions.trunk.version=trunk
+libs.stdexec.versions.trunk.path=/opt/compiler-explorer/libs/stdexec/include
```

This path will be /opt/compiler-explorer/libs/stdexec/trunk

```suggestion
libs.stdexec.versions.trunk.path=/opt/compiler-explorer/libs/stdexec/trunk/include
```